OsmIntegrator/Controllers/AccountController.cs

```diff
@@ -34,7 +35,7 @@ public class AccountController : Controller
         private readonly ILogger<AccountController> _logger;
         private readonly RoleManager<ApplicationRole> _roleManager;
         private readonly ITokenHelper _tokenHelper;
-        private readonly IValidationHelper _validationHelper;
+        private readonly IModelValidator _modelValidator;
```

Comment:
Marcin tutaj IValidationHelper zmienił się na model validator. Wynika to z faktu, że przy okazji utworzyłem walidator dla kafelków, a w przyszłości będziemy tworzyć jeszcze inne. Model validator służy do walidacji modelu, czyli tego co przychodzi w requeście.